### ISSUE TITLE
Stable release for v4.0.0

### ISSUE THREAD

**Issue by motss** [motss/app-datepicker#167]
### Done

* [x] Jumping table caption in Safari. See [Webkit bug]. (cb122ab)
* [x] `disabledDays` disables dates that do not fall in the defined weekdays.
* [x] No autofocus when first render. User can choose to focus on the first focusable element by listening to `datepicker-first-update` event. (200d0f0)
* [x] `tabindex` does not update accordingly to focused date
* [x] Update demo with latest changes
* [x] To add `alwaysResetValue` (#176)
* [x] When there is no focused date in current calendar month, keyboard key is not able to focus on calendar day since all are `tabindex="-1"` set
* [x] Provide minimal customization on scrollbar thumb in year list view (#173)
* [x] Update TypeScript to 3.8 for `import type from '...'`
* [x] Improve accessibility with VoiceOver (macOS) and NVDA (Windows). Inspired by [Angular Datepicker] and [ARIA: grid role]
* [x] Fix `mouseleave` not being handled (#174)
* [x] Support `::part` in datepicker dialog
* [x] Update tests and docs with available `part`s
* [x] Initial support for `::part`, which has been supported in stable Chrome and Firefox. See [::part()]
* [x] Fix calendar day not having consistent width thus content not being centered (#169, #172)
* [x] Add `--app-datepicker-focused-year-bg-color` (#170)
* [x] Update doc for `mwc-dialog` integration
* [x] Shoutout to `awesome-lit-html` resources to feature this element
* [x] Update `app-datepicker-dialog` to include 2 new buttons: `Clear` and `Set` (#161)
* [x] Include demo to integrate with `mwc-dialog`
* [x] Fix broken table in documentation
* [x] Update disabled day's default color in CSS Properties section

### Not under consideration

* [ ] Add tests for `mwc-dialog` integration

[::part()]: https://developer.mozilla.org/en-US/docs/Web/CSS/::part
[Angular Datepicker]: https://material.angular.io/components/datepicker/overview
[ARIA: grid role]: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Grid_Role
[Webkit bug]: https://bugs.webkit.org/show_bug.cgi?id=187903

**Comment by motss** [comment]
ETA: March 15, 2020

**Comment by motss** [comment]
📦 Released `v4` at 2a66af13acf62231630139b5e7664ac48f13812e. Closing this happily! 👍
